### PR TITLE
Prepare examples of prettier formatted code

### PR DESCRIPTION
**Pull request by JuliaGerasymenko**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+node_modules/
+applications/example/
+tests/
+schemas/
+config/
+lib/templates.js
```
[the remark this answers is not in the document]
Comment (JuliaGerasymenko) [reply .prettierignore]: Nechai wrote somewhere that .prettierignore must be similar to .eslintignore. Why is lib/templates.js ignored in eslint?

[the remark this answers is not in the document]
Comment (tshemsedinov) [reply .prettierignore]: Yes, I'll remove this line

**Comment by tshemsedinov** [comment]
Rebased on master

**Comment by tshemsedinov** [comment]
@JuliaGerasymenko please use button `resolve conversation` for fixed issues.

**Comment by tshemsedinov** [comment]
Need to rebase on master @JuliaGerasymenko

**Comment by tshemsedinov** [comment]
Not relevant for v2.x

**Pull request closed without merging**